app/javascript/bundles/HelloWorld/components/HelloWorld.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
```

Comment:
Having anything in the `app/javascript` directory would be part of strategy 2, in which we actually use webpacker.